src/main/java/com/acwong/peelplaylistcheck/playlist/PlayList.java

```diff
@@ -0,0 +1,10 @@
+package com.acwong.peelplaylistcheck.playlist;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class PlayList extends ArrayList<String> {
```

Comment:
Would you have PlayList extend ArrayList or rather use a field of type list?

Reply:
I would have PlayList to extent ArrayList.  It would serve as equivalent reference to the ArrayList.  And this can be extended if e.g. we want to introduce a label to each playlist.

In addition, for PlayLists, it would be clearer to extend ArrayList<PlayList> rather than ArrayList<List<PlayList>.